tests/integration/states/test_vmc_org_users.py

```diff
@@ -0,0 +1,54 @@
+"""
+    Integration Tests for vmc_org_users state module
+"""
+import pytest
+
+
+@pytest.fixture
+def vmc_common_data(vmc_connect):
+    data = vmc_connect.copy()
+    data.pop("vcenter_hostname")
+    data.pop("sddc_id")
+    data["hostname"] = data["authorization_host"]
+    data.pop("authorization_host")
```

Comment:
**suggestion** pop returns the value, this can be one line:
```suggestion
    data["hostname"] = data.pop("authorization_host")
```

Reply:
Updated